docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: '3.4'
+services:
+  db:
+    environment:
+      - MYSQL_USER=foreman
+      - MYSQL_PASSWORD=foreman
+      - MYSQL_DATABASE=foreman
+      - MYSQL_DATADIR_ACTION=upgrade-auto
+    hostname: db.prod.com
+    image: centos/mariadb-102-centos7
+    ports:
+      - 3306
+    restart: always
+    healthcheck:
+      test: ["CMD-SHELL", "nc -z 127.0.0.1 3306 || exit 1"]
+      interval: 30s
+      timeout: 30s
+      retries: 3
+    volumes:
+      - db:/var/lib/mysql/data
+
+  app: &app_base
+    image: quay.io/ohadlevy/foreman:containers
```

Comment:
Does this need to be adjusted?

Reply:
I'm happy to change it if poeple agree, I wanted to have a working container people can try out this pr...

Reply:
I'll adjust it once merged?

Reply:
You mean in a follow-up commit. 👍 from me.